packages/grid/src/mouse-handlers/GridTokenMouseHandler.ts

```diff
@@ -32,27 +32,30 @@ class GridTokenMouseHandler extends GridMouseHandler {
       }
     }
 
-    const { metricCalculator, renderer } = grid;
-    const metricState = grid.getMetricState();
-    const renderState = grid.getRenderState();
-    const linksInCell = metricCalculator.getTokenBoxesForVisibleCell(
+    const { renderer } = grid;
+    const renderState = grid.updateRenderState();
+    const linksInCell = renderer.getTokenBoxesForVisibleCell(
       column,
       row,
-      metricState,
-      renderer,
       renderState
     );
 
     // If there are no links in the cell, return false
-    if (linksInCell == null) {
+    if (linksInCell.length === 0) {
```

Comment:
Don't need this special case - if `linksInCell.length === 0`, we're going to skep that `for` loop anyway and set `this.currentLinkBox = undefined; return false;` anyway.